[2025-05-06T04:55:19.198+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-06T04:55:19.211+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tiktok_videos_scraper_dag.virtualenv_python manual__2025-05-06T04:55:18.921922+00:00 [queued]>
[2025-05-06T04:55:19.217+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tiktok_videos_scraper_dag.virtualenv_python manual__2025-05-06T04:55:18.921922+00:00 [queued]>
[2025-05-06T04:55:19.217+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-05-06T04:55:19.226+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): virtualenv_python> on 2025-05-06 04:55:18.921922+00:00
[2025-05-06T04:55:19.229+0000] {standard_task_runner.py:72} INFO - Started process 1886 to run task
[2025-05-06T04:55:19.232+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'tiktok_videos_scraper_dag', 'virtualenv_python', 'manual__2025-05-06T04:55:18.921922+00:00', '--job-id', '4524', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpt9solxhd']
[2025-05-06T04:55:19.233+0000] {standard_task_runner.py:105} INFO - Job 4524: Subtask virtualenv_python
[2025-05-06T04:55:19.286+0000] {task_command.py:467} INFO - Running <TaskInstance: tiktok_videos_scraper_dag.virtualenv_python manual__2025-05-06T04:55:18.921922+00:00 [running]> on host 2c7afa36055b
[2025-05-06T04:55:19.339+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='tiktok_videos_scraper_dag' AIRFLOW_CTX_TASK_ID='virtualenv_python' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T04:55:18.921922+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-06T04:55:18.921922+00:00'
[2025-05-06T04:55:19.339+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-06T04:55:19.342+0000] {process_utils.py:186} INFO - Executing cmd: /home/airflow/.local/bin/python -m virtualenv /tmp/venvkhsina00 --python=python
[2025-05-06T04:55:19.342+0000] {process_utils.py:190} INFO - Output:
[2025-05-06T04:55:20.049+0000] {process_utils.py:194} INFO - created virtual environment CPython3.10.14.final.0-64 in 235ms
[2025-05-06T04:55:20.049+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venvkhsina00, clear=False, no_vcs_ignore=False, global=False)
[2025-05-06T04:55:20.049+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/airflow/.local/share/virtualenv)
[2025-05-06T04:55:20.049+0000] {process_utils.py:194} INFO -     added seed packages: pip==25.0.1, setuptools==78.1.1, wheel==0.45.1
[2025-05-06T04:55:20.049+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-05-06T04:55:20.077+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvkhsina00/bin/pip install -r /tmp/venvkhsina00/requirements.txt
[2025-05-06T04:55:20.078+0000] {process_utils.py:190} INFO - Output:
[2025-05-06T04:55:21.111+0000] {process_utils.py:194} INFO - 
[2025-05-06T04:55:21.112+0000] {process_utils.py:194} INFO - [notice] A new release of pip is available: 25.0.1 -> 25.1.1
[2025-05-06T04:55:21.112+0000] {process_utils.py:194} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2025-05-06T04:55:21.112+0000] {process_utils.py:194} INFO - ERROR: Invalid requirement: 'playwright=1.52.0': Expected end or semicolon (after name and no valid version specifier)
[2025-05-06T04:55:21.112+0000] {process_utils.py:194} INFO -     playwright=1.52.0
[2025-05-06T04:55:21.112+0000] {process_utils.py:194} INFO -               ^ (from line 2 of /tmp/venvkhsina00/requirements.txt)
[2025-05-06T04:55:21.112+0000] {process_utils.py:194} INFO - Hint: = is not a valid operator. Did you mean == ?
[2025-05-06T04:55:21.163+0000] {taskinstance.py:3301} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 505, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 868, in execute_callable
    self._prepare_venv(tmp_path)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 769, in _prepare_venv
    prepare_virtualenv(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/python_virtualenv.py", line 119, in prepare_virtualenv
    execute_in_subprocess(pip_cmd)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/process_utils.py", line 175, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd, env=env)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/process_utils.py", line 198, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venvkhsina00/bin/pip', 'install', '-r', '/tmp/venvkhsina00/requirements.txt']' returned non-zero exit status 1.
[2025-05-06T04:55:21.171+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=tiktok_videos_scraper_dag, task_id=virtualenv_python, run_id=manual__2025-05-06T04:55:18.921922+00:00, execution_date=20250506T045518, start_date=20250506T045519, end_date=20250506T045521
[2025-05-06T04:55:21.179+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-06T04:55:21.179+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 4524 for task virtualenv_python (Command '['/tmp/venvkhsina00/bin/pip', 'install', '-r', '/tmp/venvkhsina00/requirements.txt']' returned non-zero exit status 1.; 1886)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 2995, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3149, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3173, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 505, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 868, in execute_callable
    self._prepare_venv(tmp_path)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 769, in _prepare_venv
    prepare_virtualenv(
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/python_virtualenv.py", line 119, in prepare_virtualenv
    execute_in_subprocess(pip_cmd)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/process_utils.py", line 175, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd, env=env)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/utils/process_utils.py", line 198, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venvkhsina00/bin/pip', 'install', '-r', '/tmp/venvkhsina00/requirements.txt']' returned non-zero exit status 1.
[2025-05-06T04:55:21.217+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 1
[2025-05-06T04:55:21.229+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-06T04:55:21.232+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
